Dockerfile

```diff
@@ -172,10 +185,16 @@ COPY --from=builder /usr/local/lib/python3.7/site-packages/ /usr/local/lib/pytho
 # troublesome or excessive.
 #   -trs, 15 June 2018
 COPY --from=builder \
+    /usr/local/bin/*.py \
```

Comment:
I tried running pango annotations from the ncov workflow with this line commented out and everything worked as expected.

Are there specific Python scripts we should include here? Even if there are a lot, an explicit list would help with future maintenance.

Reply:
This could have been left over from the attempt to install usher, you're right that I don't think it's necessary. I've removed it.